### ISSUE TITLE
Support filtering by "Product"

### ISSUE THREAD

**Issue by hades** [mlaitinen/ofxstatement-revolut#24]
My CSV exports from Revolut contain multiple accounts in the same file (e.g. stocks and current account). What if we added support a setting to filter the transactions by the "Product" column, so that multiple OFX files can be generated from the same CSV?

See also https://github.com/kedder/ofxstatement/issues/47

**Comment by mlaitinen** [comment]
Thanks for the feature request. I'm no longer using this plugin since YNAB now supports importing Revolut transactions through an integration. If someone is willing to implement such feature, I'd be happy to review and merge it.

As a workaround you could use [csvtool](https://github.com/maroofi/csvtool/) to filter the CSV statement files before running them through ofxstatement.

For example:
```
csvtool --encoding UTF-8 -c 2 -s '^Current$' account-statement-revolut.csv > revolut-current.csv
```
will produce a `revolut-current.csv` file that contains the header and the transactions of the current account only.

**Comment by mlaitinen** [comment]
Closed by #26
